lib/components/narrative/line-itin/connected-itinerary-body.js

```diff
@@ -0,0 +1,80 @@
+import isEqual from 'lodash.isequal'
+import TriMetLegIcon from '@opentripplanner/icons/lib/trimet-leg-icon'
+import TransitLegSummary from '@opentripplanner/itinerary-body/lib/defaults/transit-leg-summary'
+import ItineraryBody from '@opentripplanner/itinerary-body/lib/otp-react-redux/itinerary-body'
+import LineColumnContent from '@opentripplanner/itinerary-body/lib/otp-react-redux/line-column-content'
+import PlaceName from '@opentripplanner/itinerary-body/lib/otp-react-redux/place-name'
+import RouteDescription from '@opentripplanner/itinerary-body/lib/otp-react-redux/route-description'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import { showLegDiagram } from '../../../actions/map'
+import { setActiveLeg } from '../../../actions/narrative'
+import { setViewedTrip } from '../../../actions/ui'
+import TransitLegSubheader from './connected-transit-leg-subheader'
+import TripDetails from '../connected-trip-details'
+import TripTools from '../trip-tools'
+
+const noop = () => {}
+
+class ConnectedItineraryBody extends Component {
+  shouldComponentUpdate (nextProps, nextState) {
+    return !isEqual(this.props.companies, nextProps.companies) ||
+      !isEqual(this.props.itinerary, nextProps.itinerary)
+  }
+
+  render () {
+    const {
+      config,
+      diagramVisible,
+      itinerary,
+      setActiveLeg,
+      setViewedTrip,
+      showLegDiagram
+    } = this.props
+
+    return (
+      <div className='itin-body'>
+        <ItineraryBody
+          config={config}
+          diagramVisible={diagramVisible}
+          itinerary={itinerary}
+          LegIcon={TriMetLegIcon}
+          LineColumnContent={LineColumnContent}
+          PlaceName={PlaceName}
+          RouteDescription={RouteDescription}
+          setActiveLeg={setActiveLeg}
+          setLegDiagram={showLegDiagram}
+          setViewedTrip={setViewedTrip}
+          showAgencyInfo
+          showElevationProfile
+          showLegIcon
+          showMapButtonColumn={false}
+          showViewTripButton
+          toRouteAbbreviation={noop}
```

Comment:
Can we give this a comment? Not fully aware of why a noop func would need to be passed here, but that seems like a strange design.

Reply:
It's the same dilemma between `callIfValid` or passing a noop by default...

Reply:
Can't we just provide a default prop?

Reply:
See https://github.com/opentripplanner/otp-ui/pull/106.